build_tools/pkgci/build_linux_packages.sh

```diff
@@ -188,12 +188,13 @@ function build_wheel() {
 function build_iree_runtime() {
   # We install the needed build deps below for the tools.
   IREE_RUNTIME_BUILD_TRACY=ON IREE_RUNTIME_BUILD_TRACY_TOOLS=ON \
-  IREE_EXTERNAL_HAL_DRIVERS="rocm" \
+  IREE_EXTERNAL_HAL_DRIVERS="rocm" IREE_HAL_DRIVER_CUDA=ON \
   build_wheel runtime/
 }
 
 function build_iree_compiler() {
   IREE_TARGET_BACKEND_ROCM=ON IREE_ENABLE_LLD=ON \
+  IREE_TARGET_BACKEND_CUDA=ON \
```

Comment:
similar comment about one option per line (and group by category)
```suggestion
  IREE_ENABLE_LLD=ON \
  IREE_TARGET_BACKEND_ROCM=ON \
  IREE_TARGET_BACKEND_CUDA=ON \
```